### PR TITLE
Fixes for Svg.Skia

### PR DESCRIPTION
**Pull request by dansiegel**
GitHub Issue (If applicable): closes #

- fixes unoplatform/uno-private#464
- fixes unoplatform/uno-private#460
- fixes https://github.com/unoplatform/uno-private/issues/466

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Refactoring (no functional changes, no api changes)

## What is the current behavior?

- Missing the `Svg.Skia` package reference
- We calculate which packages are required twice
- We are missing the Version Override properties from the intellisense schema file

## What is the new behavior?

- Add top level dependency for `Svg.Skia` when we have the `Svg` UnoFeature
- Refactors the ImplicitPackageResolver to simply use the ProjectSystem packages that are calculated already statically. This ensure that we only calculate which packages are required once rather than introducing bugs where they may be added by the Task but not statically in the ProjectSystem targets, or vice versa.
- Adds missing Version override properties to the intellisense schema file
- Updates docs for UnoFeatures

### CODE REVIEW

**Comment by dansiegel** [comment]
@mergifyio backport release/stable/5.2

File: doc/articles/features/using-the-uno-sdk.md [hunk 1]
```diff
@@ -4,24 +4,24 @@ uid: Uno.Features.Uno.Sdk
 
 # Using the Uno.Sdk
 
-Uno Platform projects use the Uno.Sdk, and msbuild package designed to keep projects simple, yet configurable. It inherits from the `Microsoft.Net.Sdk` and `Microsoft.Net.Sdk.Web` depending on the platform.
+Uno Platform projects use the Uno.Sdk package designed to keep projects simple, yet configurable. It import the `Microsoft.Net.Sdk` (and the `Microsoft.Net.Sdk.Web` for WebAssembly).
```
Comment (agneszitte) [inline doc/articles/features/using-the-uno-sdk.md:7]:
```suggestion
Uno Platform projects use the Uno.Sdk package is designed to keep projects simple, yet configurable. It imports the `Microsoft.Net.Sdk` (and the `Microsoft.Net.Sdk.Web` for WebAssembly).
```

File: doc/articles/features/using-the-uno-sdk.md [hunk 2]
```diff
@@ -101,6 +101,9 @@ Here are the supported properties:
 - `UnoUniversalImageLoaderVersion`
 - `UnoWasmBootstrapVersion`
 
+> [!NOTE]
+> In the 5.2 version of the Uno.Sdk you must provide a value for `UnoExtensionsVersion`, `UnoThemesVersion`, `UnoToolkitVersion` and `UnoCSharpMarkupVersion` in order to use the packages associated with the UnoFeatures from these libraries as they are down stream dependencies of the Uno repository.
```
Comment (agneszitte) [inline doc/articles/features/using-the-uno-sdk.md:105]:
```suggestion
> [!NOTE]
> In the 5.2 version of the Uno.Sdk you must provide a value for `UnoExtensionsVersion`, `UnoThemesVersion`, `UnoToolkitVersion`, and `UnoCSharpMarkupVersion` in order to use the packages associated with the UnoFeatures from these libraries as they are downstream dependencies of the Uno repository.
```

File: src/SolutionTemplate/5.2/uno52AppWithLib/uno52lib/Assets/test.xml
Comment (agneszitte) [inline src/SolutionTemplate/5.2/uno52AppWithLib/uno52lib/Assets/test.xml:1]: changes needed here?

File: src/Uno.Sdk/Sdk/Sdk.props.buildschema.json [hunk 3]
```diff
@@ -141,6 +141,122 @@
...
+		"SvgSkiaVersion": {
+			"description": "Provides an explicit override for the version of Svg.Skia to use.",
+			"type": "nuget-version"
+		},
+		"UnoLoggingVersion": {
+			"description": "Provides an explicit override for the version of Uno Logging to use.",
+			"type": "nuget-version"
+		},
+		"WindowsCompatibilityVersion": {
+			"description": "Provides an explicit override for the version of Windows.Compatability to use.",
```
Comment (agneszitte) [inline src/Uno.Sdk/Sdk/Sdk.props.buildschema.json:178]:
```suggestion
			"description": "Provides an explicit override for the version of Windows.Compatibility to use.",
```

File: src/SolutionTemplate/5.2/uno52AppWithLib/uno52lib/Assets/test.xml
Comment (jeromelaban) [reply src/SolutionTemplate/5.2/uno52AppWithLib/uno52lib/Assets/test.xml:1]: No, the file needs to be there.

**Pull request merged**